suite-native/device/src/navigation/DevicePinProtectionStackNavigator.tsx

```diff
@@ -0,0 +1,24 @@
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
```

Comment:
please create new package `@suite-native/module-device-settings`, packages without `module` prefixes shoudn't navigations/screens

Reply:
There is already navigator and screen in the `@suite-native/device` package. 🤷‍♂️

Reply:
Yea, we should move it too :D